spec/controllers/omniauth_callbacks_controller_spec.rb

```diff
@@ -73,8 +73,18 @@
         allow_any_instance_of(User).to receive(:persisted?).and_return(false)
       end
 
-      it 'raises an error' do
-        expect { get_login_dot_gov }.to raise_error(ActionController::UnknownFormat)
+      it 'redirects to access-denied page' do
+        expect(get_login_dot_gov).to redirect_to('https://search.gov/access-denied')
+      end
+    end
+
+    context 'when a user is not approved' do
+      let(:email) { 'affiliate_manager_with_not_approved_status@fixtures.org' }
+      let(:user) { users(:affiliate_manager_with_not_approved_status) }
+      let(:auth) { mock_user_auth(user.email, email) }
```

Comment:
You could simplify this by omitting the `let(:email)` line (because `email` is equivalent to `user.email`), and using the default `uid` parameter in `mock_user_auth`:
```ruby
let(:auth) { mock_user_auth(user.email) }
```
Not a big deal, so I'm going to approve this and leave it up to you.

Reply:
oh yeah thats right. will do

Reply:
actually default uid has to be uniq.  and the default has already been created. I just named is not_approved12345

Reply:
👍